app/assets/v2/js/pages/bounty_details.js

```diff
@@ -358,15 +361,18 @@ window.addEventListener('load', function() {
                         }
                         actions.push(entry);
                     }
-                    if(is_on_watch_list(result['github_url'])){
+
+                    var canWatch = !isBountyOwner(result)
+
+                    if (is_on_watch_list(result['github_url']) && canWatch) {
                         var entry = {
                             href: '/unwatch',
                             text: 'Unwatch',
                             parent: 'left_actions',
                             color: 'darkGrey'
                         }
                         actions.push(entry);
-                    } else {
+                    } else if (canWatch) {
```

Comment:
seems to me that anyone shoudl be able to watch an issue.  even bountie owners.. no?

Reply:
The problem with that is that it doesn't really matter if a bounty owner is watching an issue or not. When I view the issue in the ```/explore``` dashboard, the bounty owner's issue is automatically marked with the word "Mine", so it is impossible to ever see "Watched".

See [this JavaScript snippet](https://github.com/gitcoinco/web/blob/32d8f4519e7b5b9d4133d6f1766794ba41329c4b/app/assets/v2/js/pages/dashboard.js#L174-L181) where we decide whether to generate what button to place next to a bounty on the dashboard ("Mine", "Watched", etc.). It seems this is the only place where we actually use the Watch list.

Reply:
> the bounty owner's issue is automatically marked with the word "Mine", so it is impossible to ever see "Watched".

got it; this is a UI issue that we should fix.

note that in the sidebar filters it is possible to filter only by your 'watched' issues... so there is some functional difference here though.

we should def fix the UI thing you noticed above eventually

Reply:
http://bits.owocki.com/312J1709150k/Screen%20Shot%202017-12-14%20at%208.35.22%20AM.png